csvcubed/csvcubed/utils/uri.py

```diff
@@ -79,9 +79,9 @@ def ensure_looks_like_uri(value: str) -> None:
         raise ValueError(f"'{value}' does not look like a URI.")
         
 
-def ensure_looks_like_uris(values: list[str]) -> None:
+def ensure_values_in_lists_looks_like_uris(values: list[str]) -> None:
     """
-    Ensure that :obj:`value` looks like a URI.
+    Ensure that :obj:`value` in a list looks like a URI.
```

Comment:
So here the `value` we're referring to was the name of the function's parameter which you've renamed to `values`. So something like ```Ensure that the values in a list (:obj:`values`) look like URIs.``` would work here.